.travis.yml

```diff
@@ -1,4 +1,48 @@
-sudo: enabled
-
-script:
-- exit 0
+sudo: required
+addons:
+  apt:
+    update: true
+    packages:
+    - docker-ce
+stages:
+- docker-creation
+- multiarch-release
+jobs:
+  include:
+  - stage: docker-creation
+    # For AMD64
+    script:
+    - set -e
+    - source ./.travis/travis_utils.sh
+    - prepare_everything
+    - deploy_docker './docker/amd64-Dockerfile' $KOLIBRI_AMD64 $KOLIBRI_AMD64_LATEST
+  - stage: docker-creation
+    # For arm64
+    before_script:
+    - docker run --rm --privileged multiarch/qemu-user-static:register --reset
+    script:
+    - set -e
+    - source ./.travis/travis_utils.sh
+    - prepare_everything
+    - deploy_docker './docker/arm64-Dockerfile' $KOLIBRI_arm64 $KOLIBRI_arm64_LATEST
```

Comment:
Should that be not lowercase to be right

Reply:
Haha ya. Sorry about that.
Fixed it